app/app.go

```diff
@@ -543,6 +550,13 @@ func New(
 		app.DistrKeeper,
 	)
 
+	keys[mevtypes.StoreKey] = storetypes.NewKVStoreKey(mevtypes.StoreKey)
+
+	app.MevKeeper = mevkeeper.NewKeeper(
+		appCodec,
+		keys[mevtypes.StoreKey],
+	)
```

Comment:
my understanding is that the keepers are all the things that manage the sate (e.g. keep state), which means this keeper is probably backed by a connection to the bundle submitter.

If I'm right, this should be called out. If not... What is it keeping?